changes/3986_docker_networks.yaml

```diff
@@ -0,0 +1,8 @@
+enhancement:
+  - "Support multiple docker networks with Docker Agent - [#3986](https://github.com/PrefectHQ/prefect/issues/3986)"
+
+breaking:
```

Comment:
I don't think this would be a breaking change, doesn't have any effect on users who are starting the agent strictly through the CLI 👍

Reply:
Ok, removed the `breaking`